### DISCUSSION TITLE
transform_explicit & create_dirs should not always suffix the directory structure

### DISCUSSION THREAD

**Discussion by svenXY** [jakewvincent/mkdnflow.nvim#136] [General]
Hi,

when using transform_explicit and create_dirs in combination, links are created as e.g. `[work/with/dirs](2023-01-30_work/with/dirs.md)`. I woud have expected something like `[work/with/dirs](work/with/2023-01-30_dirs.md)` as otherwise, directory hierarchies would be created over and over again.

I understand that I could change transform_explicit to do this for me properly, but imho this is so generic that it could be supported as a setting, something like `prefix_dirs=true` for the current setup and `prefix_dirs=false` to prefix the filename, but not the directory structure.

What do you think?

Thanks for the nice plugin, btw!
Sven

**Comment by jakewvincent** [comment]
Created an issue to track this request here: https://github.com/jakewvincent/mkdnflow.nvim/issues/223
